src/FSharpVSPowerTools.Logic/VSUtils.fs

```diff
@@ -87,6 +87,20 @@ type ITextBuffer with
         let span = SnapshotSpan(x.CurrentSnapshot, 0, x.CurrentSnapshot.Length)
         event.Trigger(sender, SnapshotSpanEventArgs(span))
 
+type ITextView with
+    member x.GetCaretPosition () =
+        maybe {
+          let! point = x.TextBuffer.GetSnapshotPoint x.Caret.Position
+          let line = point.Snapshot.GetLineNumberFromPosition point.Position
+          let col = point.Position - point.GetContainingLine().Start.Position
+          return (line, col)
+        }
+    member x.posAtCaretPosition () =
```

Comment:
I meant to follow naming convention: `posAtCaretPosition` --> `PosAtCaretPosition`. Is this method used at all?